config/characterizeImage.py

```diff
@@ -7,24 +7,9 @@
 
 obsConfigDir = os.path.dirname(__file__)
 
-# Cosmic rays
-# These configs match obs_subaru, to facilitate 1:1 comparisons between DECam and HSC
-config.repair.cosmicray.nCrPixelMax = 100000
-config.repair.cosmicray.cond3_fac2 = 0.4
-
 # PSF determination
 # These configs match obs_subaru, to facilitate 1:1 comparisons between DECam and HSC
 config.measurePsf.reserve.fraction = 0.2
-config.measurePsf.starSelector["objectSize"].sourceFluxField = "base_PsfFlux_instFlux"
-config.measurePsf.starSelector["objectSize"].widthMin = 0.9
-config.measurePsf.starSelector["objectSize"].fluxMin = 4000
```

Comment:
As above, these are both modified now, to:
```
# minimum width to include in histogram
config.measurePsf.starSelector['objectSize'].widthMin=0.0

# specify the minimum psfFlux for good Psf Candidates
config.measurePsf.starSelector['objectSize'].fluxMin=12500.0
```

Are these changes expected on this ticket?

Reply:
No, these are the new task defaults.

Reply:
If 0.9 / 4000 are the new task defaults, that's :+1:.